packages/core-typings/src/IVideoConference.ts

```diff
@@ -13,14 +15,36 @@ export type ConferenceInstructions = {
 };
 
 export type VideoConferenceType = DirectCallInstructions['type'] | ConferenceInstructions['type'];
+export interface IVideoConferenceUser extends Required<Pick<IUser, '_id' | 'username' | 'name'>> {
+	ts: Date;
+}
+
+export const VideoConferenceStatus = {
```

Comment:
should this be a [enum](https://www.typescriptlang.org/docs/handbook/enums.html) ?